libusb.go

```diff
@@ -224,14 +224,21 @@ func (libusbImpl) setDebug(c *libusbContext, lvl int) {
 }
 
 func (libusbImpl) getDeviceDesc(d *libusbDevice) (*DeviceDesc, error) {
-	var desc C.struct_libusb_device_descriptor
+	var desc, pathData, path = C.struct_libusb_device_descriptor{}, [8]uint8{}, []uint8{}
```

Comment:
please turn this into a var block. There is no need to intialize the vars:

```
var (
  desc C.struct_libusb_device_descriptor
  pathData [8]uint8
  path []int
)
```

Reply:
Great - will do.